stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx

```diff
@@ -39,7 +39,7 @@ const CountrySiteHeader = ({
               data={languageswitcherData.languagedata}
               headerText={languageswitcherData.headerText}
             />
-            <button className='menu-icon'></button>
+            <button className='menu-icon'>mobile bar</button>
```

Comment:
@Ashish-srij Kindly add animation for Hamburger Menu. I do not see it implemented. Kindly check the following link for the example:
https://undp-design-system.azurewebsites.net/showcase/buttons
![image](https://user-images.githubusercontent.com/94124218/147966504-b3332173-d189-488a-b30a-ca2c140ac422.png)

Reply:
@shishirphadke animation is not mentioned on requirement. Do we need to add it on lot 1.

Reply:
@shishirphadke Done.